scripts/download_wandb_run_logs.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import tqdm
+import json
+import pandas as pd
+import openvalidators
+from analysis.utils import get_runs, download_data
+
+
+DEFAULT_PROJECT = 'openvalidators'
+DEFAULT_FILTERS = {"tags": {"$in": [openvalidators.__version__]}}
+
+
+def collect_data(download_all: bool, export_path: str, wandb_run_id: str = None) -> pd.DataFrame:
+    if download_all:
+        runs = get_runs(DEFAULT_PROJECT, filters=DEFAULT_FILTERS, return_paths=True)
+        df = download_data(runs)
+        df.to_csv(export_path)
+    else:
+        if wandb_run_id is None:
+            raise Exception("Please specify a wandb run id to download")
+        else:
+            df = download_data(f'{DEFAULT_PROJECT}/{wandb_run_id}')
+            df.to_csv(export_path)
+
+    print(f'Data collected successfully at: {export_path}')
```

Comment:
Looks great! For consistency we ought to use bittensor logging instead of prints.

Reply:
I will add a TODO for that on #10